ersilia/default.py

```diff
@@ -30,6 +30,7 @@
 LOGGING_FILE = "console.log"
 CURRENT_LOGGING_FILE = "current.log"
 CARD_FILE = "card.json"
+LOCAL_CARD_FILE = "information.json"
```

Comment:
We already have a file called INFORMATION_FILE, do we need to create another one?